package.json

```diff
@@ -11,7 +11,11 @@
   },
   "dependencies": {
     "@aws-amplify/api": "^4.0.63",
+    "@aws-amplify/notifications": "1.0.12",
```

Comment:
Add a comment here to explain the new packages you added and what they’re used for

Reply:
^

Reply:
^